backend/api/views.py

```diff
@@ -1,8 +1,90 @@
-from django.shortcuts import render
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+from .serializers import FiboUserSerializer
+from .models import FiboUser
+from django.contrib.auth import authenticate
 
-# Create your views here.
-from django.http import HttpResponse
 
+@api_view(['GET'])
+def getUsers(request):
+    users = FiboUser.objects.all()
+    serializer = FiboUserSerializer(users, many=True)
+    return Response(serializer.data)
 
-def index(request):
-    return HttpResponse("Hello, world. You're at the api index.")
\ No newline at end of file
+
+@api_view(['GET'])
+def getUser(request, pk):
+    user = FiboUser.objects.get(id=pk)
+    serializer = FiboUserSerializer(user, many=False)
+    return Response(serializer.data)
```

Comment:
we should not allow _unauthenticated_ users of the API to acess whatever private, personal-identifiable, data we have saved in our database. :D

Could you please either disable this API endpoint or make sure it does not return any data for unauthenticated users?